### ISSUE TITLE
get "failure retrieving region information" in CE lite setup

### ISSUE THREAD

**Issue by danyhm** [cheat-engine/cheat-engine#2453]
Hello,

I get the above error when running CE lite setup 7.4

**Comment by cheat-engine** [comment]
Is your internet working and the installer isn't blocked?

**Comment by danyhm** [comment]
Yes my internet is working. How else would i have downloaded the installer in the first place?

**Comment by alchle** [comment]
Same issue here. Any resolution?

**Comment by alchle** [comment]
I temporary disabled my pihole and installation worked.  Must have been blocking something.

**Comment by 2mg** [comment]
I'm getting the same issue, temporarily disabled Windows firewalls and allowed the adware.

**Comment by liudonghua123** [comment]
Same problems. It's blocked by windows security. In the windows security, I permitted it, then it's ok.
